src-docs/src/views/tables/auto/auto_section.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { GuideSectionTypes } from '../../../components';
+import { renderToHtml } from '../../../services';
+import { EuiCode } from '../../../../../src/components';
+
+import { Table } from './auto';
+
+const source = require('!!raw-loader!./auto');
+const html = renderToHtml(Table);
+
+export const section = {
+  title: 'A BasicTable with auto layout',
```

Comment:
This section is no longer about just `auto` layout but layout in general
```suggestion
  title: 'Table layout',
```